### PR TITLE
iYuga0.4.0 || Changes for iYuga crashes

### PR DESCRIPTION
**Pull request by sonali-anand-tc**

### CODE REVIEW

File: Sources/iYuga/Yuga.swift [hunk 1]
```diff
@@ -351,8 +351,10 @@ public class Yuga {
         }
         let pattern = "([0-9]{2})([0-9]{2})?([0-9]{2})?"
         let m = str.groups(for: pattern)
-        let value = m[0] + ((m.count > 1) ? (":" + m[2]) : ":00")
-        valMap[pre + "time"] = value
+        if (m.count > 0) {
```
Comment (vizsatiz) [inline Sources/iYuga/Yuga.swift:354]: Is it the same in java or is the regex outputs different in swift & java ?

File: Sources/iYuga/Classifier.swift [hunk 2]
```diff
@@ -20,14 +20,15 @@ public class Classifier {
     
     public init() {}
     
-    public func getYugaTokens(_ sentence: String) -> Pair<String, Dictionary<String, AnyObject>> {
+    public func getYugaTokens(_ message: String) -> Pair<String, Dictionary<String, AnyObject>> {
         var configMap = Dictionary<String, String>()
         configMap[Constants.YUGA_CONF_DATE] = Constants.dateTimeFormatter().string(from: Date(milliseconds: 1527811200000))
-        return getYugaTokens(sentence, configMap, IndexTrack(next: 0))
+        return getYugaTokens(message + " ", configMap, IndexTrack(next: 0))
```
Comment (vizsatiz) [inline Sources/iYuga/Classifier.swift]: Isn't it better to handle not having space in `getYugaTokens` than adding space here? Doesn't it mean everyone who wants to call `getYugaTokens` have to remember this?

File: Sources/iYuga/Yuga.swift (hunk 1 above)
Comment (sonali-anand-tc) [reply Sources/iYuga/Yuga.swift:354]: Its the same but if m is empty then the application will crash. So I have added a check here. We already do this check in Java.

File: Sources/iYuga/Classifier.swift (hunk 2 above)
Comment (sonali-anand-tc) [reply Sources/iYuga/Classifier.swift]: I have removed the space as it was not needed.

**Comment by vizsatiz** [comment]
Looks good approved

**Review by vizsatiz — APPROVED** [review]

**Pull request merged**